frontend/src/components/ProjectPicker.vue

```diff
@@ -0,0 +1,191 @@
+<template>
+  <div class="container">
+    <h1 class="title">Project Picker</h1>
+
+    <div class="new-projects">
+      <router-link :to="{ name: 'NewProject' }">
+        <button class="primary-button">Create New Project</button>
+      </router-link>
+    </div>
+
+    <!-- Display projects if available -->
+    <div class="projects-grid">
+      <div v-for="project in projects" :key="project.id" class="project-card">
+        <div class="delete-button" @click.stop="deleteProject(project.id)">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="white" width="18px" height="18px">
+            <path d="M0 0h24v24H0z" fill="none"/>
+            <path d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"/>
+          </svg>
+        </div>
+        <router-link :to="{ name: 'ProjectEditor', params: { id: project.id } }" class="project-link">
+          <img :src="project.thumbnail_path" loading="lazy" :alt="'Project ' + project.id" class="project-thumbnail"/>
+          <h3 class="project-title">{{ project.id }}</h3>
+        </router-link>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref, onMounted } from "vue";
+import axios from "axios";
+
+const BACKEND_URL = import.meta.env.VITE_BACKEND_URL;
```

Comment:
Again, env imports, besonders für ne Backend URL hab ich gerne an der Stelle wo die URLs gebaut werden (aka einmal irgendwo in nem Helper)